frontend/packages/kubevirt-plugin/src/components/modals/modal/modal-footer.tsx

```diff
@@ -90,6 +96,16 @@ export const ModalFooter: React.FC<ModalFooterProps> = ({
       >
         {cancelButtonText}
       </Button>
+      {isSaveAndRestart && (
```

Comment:
While were here, can we change `LoadingInLine` into the new Patternfly Spinner?
`import { Spinner } from '@patternfly/react-core'`

Reply:
@matthewcarleton what do you think

Reply:
done.